actors/builtin/miner/miner_test.go

```diff
@@ -443,8 +447,9 @@ func TestProveCommit(t *testing.T) {
 		st := getState(rt)
 		st.LockedFunds = big.Div(st.LockedFunds, big.NewInt(2))
 		rt.ReplaceState(st)
-
-		rt.SetEpoch(precommitEpoch + miner.MaxSealDuration[st.Info.SealProofType] - 1)
+		info, err := st.GetInfo(rt.AdtStore())
+		require.NoError(t, err)
```

Comment:
Add `getInfo` to the harness. (I don't care about redundant loads of the state/info in tests).